truss/remote/baseten/service.py

```diff
@@ -0,0 +1,40 @@
+from typing import Dict
+
+from truss.remote.baseten.auth import AuthService
+from truss.remote.truss_remote import TrussService
+from truss.truss_handle import TrussHandle
+
+
+class BasetenService(TrussService):
+    # TODO(Abu): How to handle liveness/readiness probes from client?
+    def __init__(
+        self,
+        model_id: str,
+        model_version_id: str,
+        is_draft: bool,
+        truss_handle: TrussHandle,
+        auth_service: AuthService,
+        service_url: str,
+    ):
+
+        super().__init__(is_draft=is_draft, service_url=service_url)
+        self._model_id = model_id
+        self._model_version_id = model_version_id
+        self._auth_service = auth_service
+
+    @property
+    def is_live(self) -> bool:
+        return False
+
+    @property
+    def is_ready(self) -> bool:
+        return False
+
+    def predict(self, model_request_body: Dict):
+        invocation_url = f"{self._service_url}/predict"
+        response = self._send_request(invocation_url, "POST", model_request_body)
+        # TODO(Abu): Do we want to strip response to just be model output? Or keep metadata?
```

Comment:
no metadata please. it breaks the truss interface. those things should really be headers.

Reply:
it should be the same thing you'd get from calling the truss service in docker directly.

Reply:
I agree but the issue is to strip metadata out for the Baseten impl, you either 

1. Mutate the Response object such that the `_content` is now only the model output (very hacky)
2. Only return a dict, vs I think we should give users the entire Response object incl. status codes / headers / etc.

Also, can you elaborate on what you mean by "it breaks the truss interface"?

Reply:
This is a tough one. We want to avoid that wrapping but if we do that here, then it won't match what users get when hitting the Baseten hosted model via API. Comparing the two, I feel it's best to unwrap and return the model_output, to avoid propagating that wrapper. The mismatch with hitting baseten API directly could be handled through docs. I agree that in truss client it's best to mimic the Truss container's output.